Gemfile

```diff
@@ -126,6 +125,7 @@ unless ENV["APPLIANCE"]
     gem "timecop",          "~>0.7.3",  :require => false
     gem "vcr",              "~>3.0.2",  :require => false
     gem "webmock",          "~>2.3.1",  :require => false
+    gem "rails-controller-testing",     :require => false
```

Comment:
yeah, @NickLaMuro can you put it after factory_girl?  Looks good other than that.  I can easily rebase/fix any conflicts on my branch.